### PR TITLE
Nested question set routes

### LINKED ISSUE #30: Nest question sets into class_groups

### PR DESCRIPTION
**Pull request by christineoen**
closes #30

### CODE REVIEW

File: app/controllers/question_sets_controller.rb [hunk 1]
```diff
@@ -1,34 +1,42 @@
 class QuestionSetsController < ApplicationController
-  before_action :set_question_set, only: [:show, :edit, :update, :destroy]
+  #before_action :set_question_set, only: [:show, :edit, :update, :destroy]
```
Comment (gilbert) [inline app/controllers/question_sets_controller.rb]: Is there a reason this is commented out?

**Comment by gilbert** [comment]
Other than my one comment, looks good.

**Comment by cath-oneill** [comment]
I think I did that before I handed it to Christine.  I don't think I understood what it did at that point.  One thing to point out, I don't think that the json routes work on this pull request.  I might be wrong, but that's what I'm running into with the memberships routes.

**Comment by gilbert** [comment]
We'll fix the json routes when we connect it with angular. It's hard to predict exactly what we need before then.

Uncomment that line and rebase the feature branch and I'll merge it in.

**Pull request merged**